css/responsive/desktop.styl

```diff
@@ -6,6 +6,7 @@ $gutter = 32px
   
 .col-l
   padding: 0 ($gutter / 2)
+  margin-bottom: ($gutter / 2)
```

Comment:
sometimes we have rows and cols nested within rows and cols that we dont want to add extra padding and margin to. like in the case of the searchform. so `.col` just assigns it as a flex item, and `.row` assigns a flex container.  maybe we should change our class names to `.flex-container`, `.flex-item` or something so this isnt confusing?